sdv/single_table/dayz.py

```diff
@@ -1,16 +1,232 @@
+        )
+        end_datetime = pd.to_datetime(column_parameters['end_timestamp'], format=datetime_format)
+        if start_datetime > end_datetime:
+            msg = (
+                f"Invalid parameters for {column_table_msg}. The 'start_timestamp' must be "
+                "less than the 'end_timestamp'."
+            )
+            raise SynthesizerProcessingError(msg)
+    elif 'start_timestamp' in column_parameters or 'end_timestamp' in column_parameters:
+        raise SynthesizerProcessingError(
+            f'Invalid parameters for {column_table_msg}. Both a '
+            "'start_timestamp' and an 'end_timestamp' must be set."
+        )
+
+
+def _validate_categorical_parameters(column_parameters, column_table_msg):
+    if not isinstance(column_parameters.get('category_values', []), list):
+        msg = f"'category_values' for {column_table_msg} must be a list."
+        raise SynthesizerProcessingError(msg)
+
+
+def _validate_missing_value_parameters(column_parameters, column_table_msg):
+    missing_values_proportion = column_parameters['missing_values_proportion']
+    if not _is_numerical(missing_values_proportion) or (
+        missing_values_proportion < 0.0 or missing_values_proportion > 1.0
+    ):
+        msg = (
+            f"'missing_values_proportion' for {column_table_msg} "
+            'must be a float between 0.0 and 1.0.'
+        )
+        raise SynthesizerProcessingError(msg)
+
+
+def _validate_column_parameters(table, column, column_metadata, column_parameters):
+    column_table_msg = f"column '{column}' in table '{table}'"
+    sdtype = column_metadata['sdtype']
+    sdtype_parameters = SDTYPE_TO_PARAMETERS.get(sdtype, COLUMN_PARAMETER_KEYS)
+    unknown_column_parameters = column_parameters.keys() - set(sdtype_parameters)
+    if unknown_column_parameters:
+        unknown_column_parameters = "', '".join(unknown_column_parameters)
+        msg = f"The {column_table_msg} contains unexpected key(s) '{unknown_column_parameters}'."
+        raise SynthesizerProcessingError(msg)
+
+    if sdtype == 'numerical':
+        _validate_numerical_parameters(column_parameters, column_table_msg)
+    elif sdtype == 'datetime':
+        _validate_datetime_parameters(column_parameters, column_metadata, column_table_msg)
+    elif sdtype == 'categorical':
+        _validate_categorical_parameters(column_parameters, column_table_msg)
+
+    if 'missing_values_proportion' in column_parameters:
+        _validate_missing_value_parameters(column_parameters, column_table_msg)
+
+
+def _validate_table_parameters(table, table_metadata, table_parameters):
+    missing_cols = table_parameters.get('columns', {}).keys() - table_metadata.columns.keys()
+    if missing_cols:
+        missing_cols = "', '".join(missing_cols)
+        msg = (
+            f"Invalid DayZ parameters provided, column(s) '{missing_cols}' are missing from "
+            f"table '{table}'."
```

Comment:
```suggestion
            f"Invalid DayZ parameters provided, column(s) '{missing_cols}' are missing from "
            f"table '{table}' inside the metadata."
```

Or something similar